pyilluminate/illuminate.py

```diff
@@ -94,7 +130,8 @@ class Illuminate:
         '_has_autoupdate',
         '_autoclear',
         '_autoupdate',
-        '_scale_factor'
+        '_scale_factor',
+        '_thread_lock'
```

Comment:
```suggestion
        '_thread_lock',
```
This makes the diff smaller latter